[2023-10-17T11:19:20.549+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-07T00:00:00+00:00 [queued]>
[2023-10-17T11:19:20.556+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-07T00:00:00+00:00 [queued]>
[2023-10-17T11:19:20.557+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:20.565+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-04-07 00:00:00+00:00
[2023-10-17T11:19:20.571+0000] {standard_task_runner.py:57} INFO - Started process 625 to run task
[2023-10-17T11:19:20.573+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-04-07T00:00:00+00:00', '--job-id', '101', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpm5dqm1z0']
[2023-10-17T11:19:20.573+0000] {standard_task_runner.py:85} INFO - Job 101: Subtask waiting_for_data
[2023-10-17T11:19:20.607+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-07T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:20.671+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-04-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-07T00:00:00+00:00'
[2023-10-17T11:19:20.677+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:20.677+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:20.677+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:20.678+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:20.683+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210407T000000, start_date=20231017T111920, end_date=20231017T111920
[2023-10-17T11:19:20.705+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:20.715+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:42:51.560+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-07T00:00:00+00:00 [queued]>
[2023-10-18T10:42:51.571+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-07T00:00:00+00:00 [queued]>
[2023-10-18T10:42:51.572+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:42:51.625+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-04-07 00:00:00+00:00
[2023-10-18T10:42:51.633+0000] {standard_task_runner.py:57} INFO - Started process 1590 to run task
[2023-10-18T10:42:51.636+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-04-07T00:00:00+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp35cereb2']
[2023-10-18T10:42:51.637+0000] {standard_task_runner.py:85} INFO - Job 100: Subtask waiting_for_data
[2023-10-18T10:42:51.686+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-07T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:42:51.752+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-04-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-07T00:00:00+00:00'
[2023-10-18T10:42:51.771+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:42:51.774+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210407T000000, start_date=20231018T104251, end_date=20231018T104251
[2023-10-18T10:42:51.789+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 100 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1590)
[2023-10-18T10:42:51.811+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:42:51.838+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:17:25.787+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-07T00:00:00+00:00 [queued]>
[2023-10-18T11:17:25.799+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-07T00:00:00+00:00 [queued]>
[2023-10-18T11:17:25.800+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:17:25.815+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-04-07 00:00:00+00:00
[2023-10-18T11:17:25.822+0000] {standard_task_runner.py:57} INFO - Started process 438 to run task
[2023-10-18T11:17:25.825+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-04-07T00:00:00+00:00', '--job-id', '153', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmputvvoruz']
[2023-10-18T11:17:25.826+0000] {standard_task_runner.py:85} INFO - Job 153: Subtask waiting_for_data
[2023-10-18T11:17:25.876+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-07T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:17:25.966+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-04-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-07T00:00:00+00:00'
[2023-10-18T11:17:25.983+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:17:25.984+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:17:25.984+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:17:25.985+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:17:25.999+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210407T000000, start_date=20231018T111725, end_date=20231018T111725
[2023-10-18T11:17:26.039+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:17:26.133+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-19T12:22:08.805+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-07T00:00:00+00:00 [queued]>
[2023-10-19T12:22:08.818+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-07T00:00:00+00:00 [queued]>
[2023-10-19T12:22:08.819+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-19T12:22:08.837+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-04-07 00:00:00+00:00
[2023-10-19T12:22:08.844+0000] {standard_task_runner.py:57} INFO - Started process 654 to run task
[2023-10-19T12:22:08.847+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-04-07T00:00:00+00:00', '--job-id', '154', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpdcel9_cu']
[2023-10-19T12:22:08.848+0000] {standard_task_runner.py:85} INFO - Job 154: Subtask waiting_for_data
[2023-10-19T12:22:08.895+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-04-07T00:00:00+00:00 [running]> on host 6f5df7fea6fd
[2023-10-19T12:22:08.977+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-04-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-04-07T00:00:00+00:00'
[2023-10-19T12:22:08.991+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-19T12:22:08.991+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-19T12:22:08.992+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-19T12:22:08.992+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-19T12:22:09.003+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210407T000000, start_date=20231019T122208, end_date=20231019T122209
[2023-10-19T12:22:09.061+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T12:22:09.100+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
